### DISCUSSION TITLE
Congrats !

### DISCUSSION THREAD

**Discussion by Eirikr70** [breezy-weather/breezy-weather#736] [General]
I am an old user of Geometric Weather and I have recently migrated to Breezy Weather. I just wanted to congratulate the devs for the quality of the evolutions they made. That is amazing ! Especially the ability to mix the different data sources !
Keep up the good work !

**Comment by papjul** [comment]
Thanks! Kind words are always appreciated so that we don't get to see only complains x)

**Comment by malfisya** [comment]
I just want to say thank you for continuing development of this app. This is awesome!
